helpers/index.js

```diff
@@ -0,0 +1,10 @@
+const crypto = require('crypto')
+
+async function generateDeviceToken () {
+  const buffer = await crypto.randomBytes(100)
```

Comment:
See [validator for iOS and Android here](https://github.com/aerogear/aerogear-unifiedpush-server/blob/ff5ba2cb8446d929e5fd426bcb645299b4559914/model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/validation/DeviceTokenValidator.java#L31-L44)